packages/@aws-cdk/aws-sns-subscriptions/README.md

```diff
@@ -34,7 +34,9 @@ Add an HTTP or HTTPS Subscription to your topic:
 ```ts
 import subscriptions = require('@aws-cdk/aws-sns-subscriptions');
 
-myTopic.addSubscription(new subscriptions.UrlSubscription('https://foobar.com/'));
+myTopic.addSubscription(new subscriptions.UrlSubscription('https://foobar.com/', {
```

Comment:
Actually this seems covered in the `aws-sns` module docs. I'd still say lets remove the changes to this file since we don't demonstrate the usage of other subscription props here like `filterPolicy` etc.